runtime/src/accounts_db.rs

```diff
@@ -2640,13 +3058,21 @@ impl AccountsDB {
         }
     }
 
+    pub fn store_cached(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
+        self.do_store(slot, accounts, true);
+    }
+
     /// Store the account update.
     pub fn store(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
```

Comment:
nits: rename to `store_direct` or `store_uncached`?

Reply:
yeah good idea